class K2Error(Exception):
    pass

class K2SourceError(K2Error):
    pass